### ISSUE TITLE
random.uniform() returns nan when given float args

### ISSUE THREAD

**Issue by schmave** [brython-dev/brython#2063]
```python
import random
print(random.uniform(1.5, 2.5))
```

This prints `nan` on the master branch of Brython, but prints a floating point value between 1.5 and 2.5 in CPython. Thanks as always for investigating!

**Comment by austin-schick** [comment]
I noticed some unusual behavior with `random.uniform()` as well:

```python3
import random
a = random.uniform(0, 1)
print(a, type(a), a + 0)
```

on the Brython master branch, this prints something like

```
0.24037593398245927 <class 'float'> nan
```

`random.uniform()` is returning a float as expected, but actually using that float produces a `nan`.

**Comment by PierreQuentel** [comment]
I have fixed this bug, but a PR (#2066) is going to be merged with a new version of the random module.
